diff --git a/Source/Views/Edit/Plugins/ExtendedUIBehaviour.h b/Source/Views/Edit/Plugins/ExtendedUIBehaviour.h
index 694559f..18db812 100644
--- a/Source/Views/Edit/Plugins/ExtendedUIBehaviour.h
+++ b/Source/Views/Edit/Plugins/ExtendedUIBehaviour.h
@@ -146,12 +146,16 @@ class ExtendedUIBehaviour : public tracktion::UIBehaviour {
         }
 
         while (runner.isThreadRunning()) {
+#if ANDROID
+            jassertfalse;
+#else
             if (!juce::MessageManager::getInstance()->runDispatchLoopUntil(
                     10)) {
                 // Can update the GUI here if we need to, but since its just a
                 // continuous loading animation we dont need to do anything for
                 // now
             }
+#endif
         }
 
         // Once the thread finishes, hide the progress view
