diff --git a/modules/tracktion_graph/tracktion_graph/tracktion_PlayHead.h b/modules/tracktion_graph/tracktion_graph/tracktion_PlayHead.h
index 03fd5a5414..43f00f3786 100644
--- a/modules/tracktion_graph/tracktion_graph/tracktion_PlayHead.h
+++ b/modules/tracktion_graph/tracktion_graph/tracktion_PlayHead.h
@@ -185,7 +185,7 @@ private:
 inline PlayHead::PlayHead() noexcept
 {
     // There isn't currently a lock-free implementation of this on Linux
-   #if ! JUCE_LINUX && ! JUCE_WINDOWS
+   #if ! JUCE_LINUX && ! JUCE_WINDOWS && ! JUCE_ANDROID
     assert (referenceSampleRange.is_lock_free());
     assert (syncPositions.is_lock_free());
    #endif
